lua/nvim-tree/explorer/init.lua

```diff
@@ -683,6 +686,102 @@ function Explorer:get_nodes()
   return self:clone()
 end
 
+---@param new_tabpage integer
+---@return boolean
+function Explorer:is_window_event(new_tabpage)
+  local is_event_scope_window = vim.v.event.scope == "window" or vim.v.event.changed_window or false
+  return is_event_scope_window and new_tabpage == self.current_tab
```

Comment:
`self.current_tab` is good. This meets the multi-instance reqirements as the explorer is now aware of its own tab.